trade-finance-manager-ui/server/controllers/case/index.js

```diff
@@ -49,17 +49,28 @@ const getCaseDeal = async (req, res) => {
 
   let showDealCancelButton = false;
   let hasDraftCancellation = false;
+  let successMessage;
 
   if (dealCancellationIsEnabled) {
     const cancellation = await api.getDealCancellation(dealId, userToken);
     showDealCancelButton = canDealBeCancelled(cancellation.status);
     hasDraftCancellation = isDealCancellationInDraft(cancellation.status);
+
+    if (cancellation.status === TFM_DEAL_CANCELLATION_STATUS.COMPLETED) {
+      successMessage = getFlashSuccessMessage(req);
```

Comment:
This is tying our implementation quite a lot to cancellation. Thinking ahead to abandonment we'll want this same success message to be displayed (and in general it might be useful to use this success message for other features. Maybe we need 
```
let successMessage = getFlashSuccessMessage(req);
```
higher up, and this is overwritten if the cancellation is scheduled (though this could also provide some fun problems down the line :/

Reply:
I think a more scalable approach would be to extract the `successMessage` logic into a helper and the controller just consumes the helper.

Not for this PR though I think we should keep it simple atm, but if/when there is a 3rd possible success message we should think about a better way and reducing logic/conditions in the controller

Also with such a helper, would not want to pass around `req` (bad practice), one for another day

Reply:
Good idea, I've done exactly that now - I think that is better and will work automatically for abandonments.